functions/src/services/user/getAccountStats.ts

```diff
@@ -8,7 +8,10 @@ const githubStats = (user: User, social: UserSocial): UserSocial | null => {
       github: {
         following: user.github.additionalUserInfo.profile.following,
         followers: user.github.additionalUserInfo.profile.followers,
-        updatedAt: (user.github.additionalUserInfo.profile.updatedAt ? user.github.additionalUserInfo.profile.updatedAt : new Date().toISOString())
+        updatedAt: (user.github.additionalUserInfo.profile.updatedAt ? user.github.additionalUserInfo.profile.updatedAt : new Date().toISOString()),
```

Comment:
I think `Date` for Firestore has been deprecated and they recommend using their library to create the date